config/itemTypes.json

```diff
@@ -903,6 +903,38 @@
     "id": "Pistol",
     "name": "Pistol"
   },
+  {
+    "id": "/Lotus/Weapons/Grineer/KuvaLich/Secondaries/",
+    "name": "Pistol"
+  },
+  {
+    "id": "/Lotus/Weapons/VoidTrader/VTDetron",
+    "name": "Pistol"
+  },
```

Comment:
Detron should be classified as a shotgun or as a pistol? https://wiki.warframe.com/w/Detron https://wiki.warframe.com/w/Mara_Detron